client/src/pages/SignUp/SignUp.tsx

```diff
@@ -38,24 +37,23 @@ export default function Register(): JSX.Element {
 
   return (
     <Grid container component="main" className={classes.root}>
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} elevation={6} component={Paper} square>
         <Box
           display="flex"
           justifyContent="space-between"
           alignItems="flex-start"
           flexDirection="column"
           className={classes.authWrapper}
         >
-          <AuthHeader linkTo="/login" asideText="Already have an account?" btnText="Login" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Create an account
-                </Typography>
-              </Grid>
-            </Grid>
+          {/* <AuthHeader linkTo="/login" asideText="Already have an account?" btnText="Login" /> */}
+          <Box width="100%" maxWidth={450} p={3} alignSelf="center" elevation={6} component={Paper} square>
             <SignUpForm handleSubmit={handleSubmit} />
+            <Box p={2} alignSelf="center" style={{ textAlign: 'center' }}>
+              <span>I already have an account </span>
+              <Link to="/login" style={{ display: 'inline', color: '#f14140' }}>
+                Login
+              </Link>
+            </Box>
```

Comment:
We should probably make this a component though since it's used in Login page as well.